app/web/service/donations.ts

```diff
@@ -11,3 +12,8 @@ export async function initiateDonation(amount: number, recurring: boolean) {
   const res = await client.donations.initiateDonation(req);
   return res.getStripeCheckoutSessionId();
 }
+
+export async function getDonationPortalLink() {
+  const res = await client.donations.getDonationPortalLink(new Empty());
+  return res.getStripePortalUrl();
```

Comment:
We normally define the req and then pass that to the res no?

Reply:
You mean `const req = new Empty();`? I'm seeing both when searching for `new Empty()` in `//app/web`

Reply:
Oh I see now there is no request type in the protos.